pkg/logproto/logproto.proto

```diff
@@ -56,4 +56,4 @@ message Stream {
 message Entry {
   google.protobuf.Timestamp timestamp = 1 [(gogoproto.stdtime) = true, (gogoproto.nullable) = false, (gogoproto.jsontag) = "ts"];
   string line = 2 [(gogoproto.jsontag) = "line"];
-}
```

Comment:
:-(

Reply:
Yep forgot to `git config --global core.autocrlf input` on my new station. Will do.

Reply:
Alright this is gone.